examples/fork/fork.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    execute(true);
+    runtime:sleep(1);
+    execute(false);
+}
+
+function execute(boolean isForked) {
```

Comment:
Do we really need a separate function?

Reply:
I'm demonstrating conditional cases here. That's why, I'm having a separate function and calling it twice

Reply:
IMO, calling it twice and showing doesn't really add value here.

Reply:
I will update that